app/bundles/NotificationBundle/Form/Type/NotificationConfigType.php

```diff
@@ -18,10 +18,6 @@
  */
 class NotificationConfigType extends AbstractType
 {
-    /**
-     * @param FormBuilderInterface<int,FormBuilderInterface> $builder
-     * @param array<string,mixed>                            $options
```

Comment:
I'd leave the array PHPDoc, will need it later in phpstan level.

Reply:
Can you help me with the PHPSTAN errors this generates?
```
  Line   app/bundles/NotificationBundle/Form/Type/NotificationConfigType.php                                                                                                                               
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  25     Generic type Symfony\Component\Form\FormBuilderInterface<int, Symfony\Component\Form\FormBuilderInterface> in PHPDoc tag @param for parameter $builder specifies 2 template types, but interface  
         Symfony\Component\Form\FormBuilderInterface supports only 1: TData                                                                                                                                
  25     Parameter #1 $builder (Symfony\Component\Form\FormBuilderInterface<int, Symfony\Component\Form\FormBuilderInterface>) of method                                                                   
         Mautic\NotificationBundle\Form\Type\NotificationConfigType::buildForm() should be compatible with parameter $builder (Symfony\Component\Form\FormBuilderInterface<array|null>) of method          
         Symfony\Component\Form\AbstractType<array>::buildForm()                                                                                                                                           
  25     Parameter #1 $builder (Symfony\Component\Form\FormBuilderInterface<int, Symfony\Component\Form\FormBuilderInterface>) of method                                                                   
         Mautic\NotificationBundle\Form\Type\NotificationConfigType::buildForm() should be compatible with parameter $builder (Symfony\Component\Form\FormBuilderInterface<array|null>) of method          
         Symfony\Component\Form\FormTypeInterface<array>::buildForm()    
```
I didn't find any other place that would have these PHPDocs

Reply:
If I do so I get 
```
 Line   app/bundles/ConfigBundle/Form/Type/DsnType.php                                                                                         
 ------ --------------------------------------------------------------------------------------------------------------------------------------- 
  34     Method Mautic\ConfigBundle\Form\Type\DsnType::buildForm() has parameter $builder with no value type specified in iterable type array.  
         💡 See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type   
 ```
I think the types should be defined in the parent method in Symfony rather than in 245 places that are overwriting this method in Mautic.

Reply:
Well, then leave as it was initially in your commit. That'll require a more targeted refactoring. FYI: that phpstan rule targets all form types to identify what object/array is in the form and notify of missing/invalid fields